cloudinstall/juju.py

```diff
@@ -40,6 +40,31 @@ def __init__(self, juju):
         self._juju_status = None
         self.valid_states = ['pending', 'started', 'down']
 
+    def get_agents_states(self):
```

Comment:
super minor, take it or leave it:
not that we'll ever actually say this out loud, but if you do, it's awkward. calling this `get_agent_states` sounds better to me